packages/telemetry/src/telemetry.ts

```diff
@@ -4,12 +4,15 @@ import * as os from 'os';
 import { sync as readPackageUpSync } from 'read-pkg-up';
 import { TelemetryClient, setup as AppInsights } from 'applicationinsights';
 import Conf from 'conf';
-import { ChildProcess, exec as execCallback, spawn as spawnCallback } from 'child_process';
+import { exec as execCallback } from 'child_process';
 import { promisify } from 'util';
 import { PathLike } from 'fs';
+import { CommandReturn } from '@appland/appmap/src/cmds/agentInstaller/commandStruct';
+import CommandStruct from '@appland/appmap/src/cmds/agentInstaller/commandStruct';
+import { run } from '@appland/appmap/src/cmds/agentInstaller/commandRunner';
```

Comment:
Isn't `commandStruct` part of this package now?

Reply:
Same as above, the symlink should be in `src/`.